test/compilable/traits.d

```diff
@@ -35,6 +35,15 @@ version (linux)
 
 static assert(__traits(getTargetInfo, "cppStd") == 199711);
 
+enum testPredefinedVersions = ()
+{
+    static foreach (predef; __traits(getTargetInfo, "predefinedVersions"))
+        mixin("enum __"~predef~"__ = 1;");
```

Comment:
These are reserved. Add a more specific name to avoid future collisions and therefore change this test case.